tmt/steps/execute/internal.py

```diff
@@ -34,7 +34,7 @@
 TEST_PIDFILE_LOCK_FILENAME = f'{TEST_PIDFILE_FILENAME}.lock'
 
 #: The default directory for storing test pid file.
-TEST_PIDFILE_ROOT = Path('/var/tmp')  # noqa: S108 insecure usage of temporary dir
+TEST_PIDFILE_ROOT = Path('/var/tmp/pid')  # noqa: S108 insecure usage of temporary dir
```

Comment:
What about using `/var/tmp/tmt/pid` instead to have all `tmt` releated files under one directory?

Reply:
addressed